package.json

```diff
@@ -10,7 +10,7 @@
     "conc": "./dist/bin/concurrently.js"
   },
   "engines": {
-    "node": "^12.20.0 || ^14.13.0 || >=16.0.0"
+    "node": ">=14.13.0"
```

Comment:
There was a reason that v15 was skipped altogether:
https://github.com/open-cli-tools/concurrently/blob/d41e78d75af2bd5d32043e0600071963f8610494/index.mjs#L7-L8

Reply:
Oh, thanks! Sorry, haven't thought about v15. Fixed in #392.